linux/ibus-keyman/src/test/meson.build

```
@@ -0,0 +1,66 @@
+keymanutil_sources = [
+  'keymanutil_tests.c',
+  util_files,
+]
+
+keymanutil_deps = [ibus, gtk, x11, json_glib, kmnkbp_lib]
+
+test_include_dirs = [
+  include_dirs,
+  include_directories('..'),
+  include_directories('@0@/..'.format(meson.current_build_dir())),
+]
+
+executable(
+  'keymanutil-tests',
+  sources: keymanutil_sources,
+  dependencies: keymanutil_deps,
+  include_directories : test_include_dirs
+)
+
```

```suggestion

test_env = [
  'G_TEST_SRCDIR=' + meson.current_source_dir(),
  'G_TEST_BUILDDIR=' + meson.current_build_dir(),
]

```

Adding a variable for the repeated env; also, I think in things like this, string concatenation is easier to read.